src/test/socket/socketpair/socketpair.shadow.config.yaml

```diff
@@ -0,0 +1,32 @@
+topology:
```

Comment:
Need to delete the `src/test/socket/socketpair/socketpair.shadow.config.xml` file.